### ISSUE TITLE
Connecting to fulcro-inspect standalone from an electron app

### ISSUE THREAD

**Issue by mruzekw** [fulcrologic/fulcro#420]
**Goal: Connect an fulcro app in an Electron instance to the standalone fulcro-inspect**

The crux of the issue is that the fulcro app is trying to connect to the fulco-inspect instance via a `file://` protocol while a version of a supporting library doesn't honor the host param.

**Problems:**

1. `[com.taoensso/sente "1.15.0"]` doesn't respect the host param to `make-channel-socket-client!` and tries to default to the window host name which ends up in trying to connect with a URL like:
```
ws://:8237/chsk?client-id=37c0ef8f-691c-470c-82e2-98c841816f71&csrf-token=no-token-desired
```

2. `com.taoensso/sente` automatically tries the location protocol if it's not given one. Resulting in an error like this:

```
Error: Invariant violation in `taoensso.sente:1453`. Test form `([:el #{"https:" "http:"}] protocol)` failed against input val `"file:"`
```

**Solution:**

1. Upgrade to `[com.taoensso/sente "1.16.0-alpha1"]` in Fulcro or document somewhere this dep needs to be overwritten during dev.
2. Explicitly pass a protocol to `make-channel-socket-client!` in Fulcro

Happy to submit a PR for this, unless there's a better alternative.

**Alternatives:**

1. Use `electron-devtools-installer`

One I tried was using https://github.com/MarshallOfSound/electron-devtools-installer in my main script. But for some reason the fulcro-inspect tools didn't want to download.

The error being:

```
Invalid header: Does not start with Cr24
```

2. Manually load the fulcro-inspect extension into the Electron app

Another alternative would be to document how to download and add the extension manually or make a starter template people can go off of.

This would be the most arduous path for users in my opinion.

See: https://www.electronjs.org/docs/tutorial/devtools-extension#how-to-load-a-devtools-extension

**Comment by awkay** [comment]
Hey. Glad to have a PR. Upgrading Sente is great, as react native needs it too.

Just be careful of the protocol setting in m-c-s-c, since F.I. electron should be usable with webapps as well (some ppl don't want to use Chrome at all).  I think it would be best to figure out if `file` is going to be used, and *only* fix it in that case (letting sente figure it out otherwise, since that helps solve http/https diffs as well).

**Comment by mruzekw** [comment]
Have a PR up! Did exactly what you suggested. Let me know if you'd like any changes! Thanks!

**Comment by awkay** [comment]
Fixed in pr #422
